contributed_definitions/NXregion.nxdl.xml

```diff
@@ -0,0 +1,193 @@
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition 
+	category="base" 
+	xmlns="http://definition.nexusformat.org/nxdl/3.1"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+	name="NXregion" 
+	type="group" extends="NXobject">
+	<symbols>
+		<doc>
+			These symbols will denote how the shape of the parent group's data field,
+			
+			.. math:: (D_0, ..., D_{\mathbf{O}-1}, d_0, ..., d_{\mathbf{R}-1})
+			
+			could be split into a left set of **O** outer dimensions, :math:`\boldsymbol{D}`,
+			and a right set of **R** region dimensions, :math:`\boldsymbol{d}`,
+			where the data field has rank **O** + **R**. Note **O** :math:`>= 0`.
+		</doc>
+		<symbol name="O"><doc>Outer rank</doc></symbol>
+		<symbol name="R"><doc>Region rank</doc></symbol>
+	</symbols>
+	<doc>
+		Geometry and logical description of a region of data in a parent group. When used, it could be a child group to, say, :ref:`NXdetector`.
+
+		This can be used to describe a subset of data used to create downsampled data or to derive
+		some data from that subset.
+
+		Note, the fields for the rectangular region specifiers follow HDF5’s dataspace hyperslab parameters
+		(see https://portal.hdfgroup.org/display/HDF5/H5S_SELECT_HYPERSLAB). Note when **block** :math:`= 1`,
+		then **stride** :math:`\equiv` **step** in Python slicing.
+		
+		For example, a ROI sum of an area starting at index of [20,50] and shape [220,120] in image data::
+
+			detector:NXdetector/
+				data[60,256,512]
+				region:NXregion/
+					@type  = "rectangular"
+					parent = "data"
+					start  = [20,50]
+					count  = [220,120]
+					statistics:NXdata/
+						@signal = "sum"
+						sum[60]
+
+		the ``sum`` dataset contains the summed areas in each frame.
+		Another example, a hyperspectral image downsampled 16-fold in energy::
+
+			detector:NXdetector/
+				data[128,128,4096]
+				region:NXregion/
+					@type  = "rectangular"
+					parent = "data"
+					start  = [2]
+					count  = [20]
+					stride = [32]
+					block  = [16]
+					downsampled:NXdata/
+						@signal = "maximum"
+						@auxiliary_signals = "copy"
+						maximum[128,128,20]
+						copy[128,128,320]
+
+		the ``copy`` dataset selects 20 16-channel blocks that start 32 channels apart,
+		the ``maximum`` dataset will show maximum values in each 16-channel block
+		in every spectra.
+	</doc>
+	<attribute name="type" optional="false">
+		<doc>
+			This is ``rectangular`` to describe the region as a hyper-rectangle in
+			the index space of its parent group's data field
+		</doc>
+		<enumeration>
+			<item value="rectangular" />
+		</enumeration>
+	</attribute>
+	<field name="parent">
+		<doc>
+			The name of data field in the parent group or the path of a data field relative
+			to the parent group (so it could be a field in subgroup)
```

Comment:
May add some additional clarification?

```
The name of data field in the parent group or the path of a data field relative
to the parent group (so it could be a field in a subgroup of the parent group).
The rank of this data field is **O** + **R**.
```

As you prefer.

Reply:
Thanks!